app/models/cart.rb

```diff
@@ -32,4 +32,20 @@ def total
     end
   end
 
+	def limit_reached?(item_id)
+		@contents[item_id] == Item.find(item_id).inventory
+        end
+
```

Comment:
This looks like the stopping point for incrementation, right? Just going and checking inventory directly.

Reply:
yeah.  its handled by like 4 methods in the model, but they are just built around a commented out method from the instructors, so it should be the correct approach